generator/mavgen.py

```diff
@@ -21,8 +21,8 @@
 schemaFile = os.path.join(os.path.dirname(os.path.realpath(__file__)), "mavschema.xsd")
 
 # Set defaults for generating MAVLink code
-DEFAULT_WIRE_PROTOCOL = mavparse.PROTOCOL_1_0
-DEFAULT_LANGUAGE = 'Python'
+DEFAULT_WIRE_PROTOCOL = mavparse.PROTOCOL_2_0 ##mavparse.PROTOCOL_1_0
```

Comment:
These bits should not be changed without general agreement. Not relevant to fix, and not something all users would be happy with